dependent/academic-psychiatry.csl

```diff
@@ -2,17 +2,15 @@
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
   <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
-    <title>Modern Rheumatology</title>
-    <title-short>Mod Rheumatol</title-short>
-    <id>http://www.zotero.org/styles/modern-rheumatology</id>
-    <link href="http://www.zotero.org/styles/modern-rheumatology" rel="self"/>
+    <title>Academic Psychiatry</title>
+    <id>http://www.zotero.org/styles/academic-psychiatry</id>
+    <link href="http://www.zotero.org/styles/academic-psychiatry" rel="self"/>
```

Comment:
GitHub or git mistakenly thinks that we renamed modern-rheumatology.csl to academic-psychiatry.csl, while in reality we deleted modern-rheumatology.csl and created academic-psychiatry.csl. I'm not sure if there is an easy solution to this.